src/platform/EFR32/efr32-chip-mbedtls-config.h

```diff
@@ -75,6 +75,7 @@ typedef void mbedtls_ecp_restart_ctx;
 #define MBEDTLS_CTR_DRBG_C
 #define MBEDTLS_ECDH_LEGACY_CONTEXT
 #define MBEDTLS_ECDSA_C
+#define MBEDTLS_ECDSA_DETERMINISTIC
```

Comment:
Just curious why did this change land in this PR?

Reply:
This is required by openthread when you enable srp client.